### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

**Review by Enrique-C — APPROVED** [review]

File: trello/api/features/steps/steps.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Here we are importing all generic steps from core
+from core.api.features.steps.Common import *
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/steps/steps.py:2]: delete ***** and add the needed libraries or dependencies.

File: core/api/features/steps/Common.py [hunk 2]
```diff
@@ -1,29 +1,43 @@
-from behave import step
-
+from behave import given, when, then, step
+from core.api.request_api.RequestApi import RequestApi
 from core.utils.SchemaValidator import SchemaValidator
 
 
-@step('Sets a "{method}" request to "{endpoint}"')
+@given('Sets a "{method}" request to "{endpoint}"')
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py]: this is step

File: core/api/features/steps/Common.py [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
 def step_impl(context, method, endpoint):
     context.endpoint_value = endpoint
     context.data_table = context.table
     context.method = method
+    context.response_param = {}
+    context.request_api = RequestApi()
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py]: update this lines of code

File: core/api/features/steps/Common.py [hunk 4]
```diff
@@ -36,4 +35,4 @@ def step_impl(context, schema):
 def step_impl(context):
     """
     :type context: behave.runner.Context
-    """
+    """
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py]: add a new line at end of this file

File: core/api/features/steps/Common.py [hunk 5]
```diff
@@ -1,5 +1,4 @@
 from behave import step
-
 from core.utils.SchemaValidator import SchemaValidator
```
Comment (JuanCarlosMartinez4) [inline core/api/features/steps/Common.py]: add new line above this one.

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**